### PR TITLE
Add allcontributors setup

### LINKED ISSUE #371: Integrate with https://allcontributors.org?
Looks like a nice approach.

### PR DESCRIPTION
**Pull request by snowman2**
- [x] Closes #371

I saw this at the SciPy conference and I liked the concept. I tried my best to recognize contributions, but probably missed some. If I did, please let me know. Also, if you would prefer not to be recognized on the README, please let me know.

Tag:
@jswhit @micahcochran @jorisvandenbossche @cjmayo @cffk @zippy1981 @jdkloe @georgeouzou @djhoese @mitkin @dopplershift @artttt @ocefpaf @heitorPB @sebastic @nickeubank @mdunphy @matthew-brett @jdemaeyer @bmwiedemann @ReallyNiceGuy @ChrisBarker-NOAA @kbevers @rouault @cgohlke @chrrrisw

### CODE REVIEW

**Comment by djhoese** [comment]
So does the bot automatically update the README for this service?

**Comment by snowman2** [comment]
> So does the bot automatically update the README for this service?

From what I can tell from the docs, it will create a PR when you ping it.

File: .all-contributorsrc [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+      "name": "Marco Aurélio da Costa",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/6545730?v=4",
+      "profile": "https://github.com/ReallyNiceGuy",
+      "contributions": [
+        "code"
+      ]
+    },
+    {
+      "login": "ChrisBarker-NOAA",
+      "name": "Chris Barker",
```
[the remark this answers is not in the document]
Comment (snowman2) [reply .all-contributorsrc]: Thanks for that. Just made the change.

[the remark this answers is not in the document]
Comment (ChrisBarker-NOAA) [reply .all-contributorsrc]: Thanks!

**Pull request merged**